lib/css/CssParser.js

```diff
@@ -335,10 +379,44 @@ class CssParser extends Parser {
 			string: (input, start, end) => {
 				switch (mode) {
 					case CSS_MODE_AT_IMPORT_EXPECT_URL: {
-						modeData.url = cssUnescape(input.slice(start + 1, end - 1));
+						modeData.url = normalizeUrl(input.slice(start + 1, end - 1), true);
 						mode = CSS_MODE_AT_IMPORT_EXPECT_SUPPORTS;
 						break;
 					}
+					default: {
+						// TODO move escaped parsing to tokenizer
+						const lastFunction = functionStack[functionStack.length - 1];
+
+						if (
+							lastFunction &&
+							(lastFunction[0].replace(/\\/g, "").toLowerCase() === "url" ||
+								/^(-\w+-)?image-set$/i.test(lastFunction[0].replace(/\\/g, "")))
```

Comment:
`/^(-\w+-)?image-set$/i` appears to be stateless. Let's hoist it and give it a helpful/descriptive name.